lib/tempus.ex

```diff
@@ -208,7 +211,10 @@ defmodule Tempus do
         %Tempus.Slot{from: nil, to: ~U[2020-08-06 23:59:59.999999Z]}
       ]
   """
-  def next_free(%Slots{slots: slots}, opts \\ []),
+  def next_free(slots, opts \\ [])
+
+  @telemetria level: :info
+  def next_free(%Slots{slots: slots}, opts),
```

Comment:
I don't understand this. Can you explain?

Reply:
You don’t understand exactly _what_?

Reply:
I've never seen this before. So `def next_free(slots, opts \\ [])` actually calls `def next_free(%Slots{slots: slots}, opts)` without any explicitly stating it. AFAIC see just no `, do:` is a green light 👍

Reply:
It’s a so-called “function header.”

It is _mandatory_ when you have [several different clauses with optional parameters](https://stackoverflow.com/a/41357084/2035262).

In this case, I overcome a bug [not yet implemented feature] in `telemetria` that fails to understand default values in the annotated functions. :facepalm: 

I will fix it soon, but meanwhile, this workaround works pretty well.